### PR TITLE
Contribution Guide

### PR DESCRIPTION
**Pull request by fellyph**
## Description

This pull request contains helpful information for those who want to collaborate with the project. 
- Introduction
- How to research
- Creating pull requests
- Organizing the cookie information

### CODE REVIEW

**Comment by fellyph** [comment]
@gagan0123 @milindmore22 @jkwakman 

Could you review this first version.

**Comment by milindmore22** [comment]
@fellyph can we please also include screenshots, you can upload them on https://snipboard.io/

**Comment by fellyph** [comment]
@milindmore22 I have included the screenshots.

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+## Contributing to the repository
+To contribute to the project, you will need a [GitHub account](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github) for version control and a text editor like [VS Code](https://code.visualstudio.com/) or a similar editor for the CSV file.
+
+To improve the readability of the CSV file, some code editors, such as Visual Code, have extensions to provide a better visualization from the CSV file:
+
+- [Edit CSV](https://marketplace.visualstudio.com/items?itemName=janisdd.vscode-edit-csv)
+- [CSV rainbow](https://marketplace.visualstudio.com/items?itemName=mechatroner.rainbow-csv)
+- [CSV to table](https://marketplace.visualstudio.com/items?itemName=phplasma.csv-to-table)
+
+You can fork the GitHub project to start editing and creating your branches. Once you've filled in the information about the cookie, you'll be required to make a pull request to be evaluated by the repository owner. In the following section, we will explain step-by-step how you can set up the project.
```
Comment (milindmore22) [inline docs/CONTRIBUTING.md:53]:
```suggestion
You can fork this GitHub repo to start editing and creating your branches. Once you've filled in the information about the cookie, you'll be required to make a pull request to be evaluated by the repository owner. In the following section, we will explain step-by-step how you can set up the project.
```

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+## Setup repository on your local
+Once you have your GitHub account, you must fork the Open Cookie Database repository. This step will create a repository with the same code and visibility in your account.
+
+To fork, click on the fork button in the top-right corner of the upstream repository.
+
+<img src="https://i.ibb.co/XsdGHQ9/github-repo-open-cookie-db.png" alt="Open cookie database repository" width="640">
+
+Once your fork is ready, you must clone that repository to your computer. You can do this using the git command, or if you are not tech-savvy, you can use the GitHub application, which provides a GUI for ease of use.
```
Comment (milindmore22) [inline docs/CONTRIBUTING.md:62]:
```suggestion
Once your fork is ready, you must clone that repository to your computer. You can do this using the git command, or if you are not tech-savvy, you can use the [GitHub application](https://desktop.github.com/), which provides a GUI for ease of use.
```

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+3. Open the file open-cookie-database.csv and start to contribute.
+
+## Editing
+Once you are ready, remember to create a branch, for example, `cookie-[vendor-name]` to submit your changes and start editing the file. The first information is a UUID to register the new cookie. A unique ID creates this information. You can find it online at websites like [uuid generator](https://www.uuidgenerator.net/) or use the node package UUID: `$ npx uuid`. 
+
+Add the related information to the cookie, and two fields could have special attention. The cookie name could have a dynamic ID, for example, `cookie_[site-id]`. If this applies to your case, the last property on the cookie is a wildcard. For wildcard cookies, set the value to 1 instead; the dynamic property doesn’t need to be specified. For example, we set the previous cookie as `cookie_`.
+
+<img src="https://i.ibb.co/Nn63CmP/open-cookie-db-vscode.png" alt="Open Cookie database on VS Code" width="640">
+
+In another scenario, some solutions set the cookies as first-party; for this scenario, it can be empty.
```
Comment (milindmore22) [inline docs/CONTRIBUTING.md:83]: Can we please also add? `How to organize the cookies` sub section

Comment (fellyph) [reply docs/CONTRIBUTING.md:83]: Did you pull the latest version? Now, is called Organizing Cookies

**Review by milindmore22 — APPROVED** [review]
LGTM

**Comment by jkwakman** [comment]
Document is perfect now. Merging pull request!

**Pull request merged**